### ISSUE TITLE
About .state File

### ISSUE THREAD

**Issue by ismetatabay** [ewrfcas/Face-Super-Resolution#23]
Hello,

First of all, thank you for your work. There is an issue I should consult with you. I want to do a training to fit animated characters using trained weights, but I guess I need a .state file for that. Is it possible for me to access this file?

Thank you in advance, I wish you a good day.

**Comment by ewrfcas** [comment]
If you want to finetune the model, you can load generator and discriminator weights. Unfortunately, the state file was lost because it is an old project. But some related information I have offered in the README, such as you can start from xxxx iters (the file name before '_') or use what learning rate (lr_steps=50000, decay=0.5).
